### PR TITLE
add tagging ibm: hardware, vm

### PR DESCRIPTION
**Pull request by athiruma**

### CODE REVIEW

File: tests_requirements.txt [hunk 1]
```diff
@@ -5,4 +5,6 @@ moto==2.3.2
 pandas
 requests==2.27.1
 typeguard==2.13.3
+SoftLayer==6.1.1
+python-ldap==3.4.2
```
Comment (ebattat) [inline tests_requirements.txt:9]: python-ldap==3.4.2, twice, delete one ?

File: requirements.txt [hunk 2]
```diff
@@ -14,4 +14,6 @@ sphinx-rtd-theme==1.0.0
 google-api-python-client==2.57.0
 google-auth-httplib2==0.1.0
 google-auth-oauthlib==0.5.2
+SoftLayer==6.1.1
+python-ldap==3.4.2
```
Comment (ebattat) [inline requirements.txt:18]: python-ldap==3.4.2, twice, delete one ?

Comment (ebattat) [reply requirements.txt:18]: Missing SoftLayer==6.1.1 in setup.py

File: cloud_governance/policy/ibm/tag_baremetal.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
+
```
Comment (ebattat) [inline cloud_governance/policy/ibm/tag_baremetal.py:1]: I think we need to add it to documentation

Comment (athiruma) [reply cloud_governance/policy/ibm/tag_baremetal.py:1]: yes, ok.

File: cloud_governance/common/clouds/ibm/account/ibm_account.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import SoftLayer
+import pandas as pd
+
+
+from cloud_governance.common.google_drive.google_drive_operations import GoogleDriveOperations
+from cloud_governance.common.ldap.ldap_search import LdapSearch
+
+
+class IBMAccount:
+
```
Comment (ebattat) [inline cloud_governance/common/clouds/ibm/account/ibm_account.py:12]: Missing Documentation

File: cloud_governance/ibm/ibm_operations/ibm_operations.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import os
+
+
+class IBMOperations:
+
```
Comment (ebattat) [inline cloud_governance/ibm/ibm_operations/ibm_operations.py:5]: Missing Documentation

File: cloud_governance/ibm/ibm_operations/tag_resources.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import importlib
+import inspect
+
+from cloud_governance.common.logger.init_logger import logger
+from cloud_governance.ibm.ibm_operations.ibm_operations import IBMOperations
+
+
+class TagResources(IBMOperations):
```
Comment (ebattat) [inline cloud_governance/ibm/ibm_operations/tag_resources.py:8]: Missing Documentation

File: cloud_governance/ibm/tagging/tagging_operations.py [hunk 7]
```diff
@@ -0,0 +1,57 @@
+import os
+from ast import literal_eval
+
+from cloud_governance.common.clouds.ibm.account.ibm_account import IBMAccount
+from cloud_governance.common.clouds.ibm.classic.classic_operations import ClassicOperations
+
+
+class TaggingOperations:
```
Comment (ebattat) [inline cloud_governance/ibm/tagging/tagging_operations.py:8]: Missing Documentation

**Comment by ebattat** [comment]
@athiruma, did you add documentation to docs ?

File: tests_requirements.txt (hunk 1 above)
Comment (athiruma) [reply tests_requirements.txt]: fixed

File: requirements.txt (hunk 2 above)
Comment (athiruma) [reply requirements.txt]: fixed

File: cloud_governance/common/clouds/ibm/account/ibm_account.py (hunk 4 above)
Comment (athiruma) [reply cloud_governance/common/clouds/ibm/account/ibm_account.py:12]: done

File: cloud_governance/ibm/ibm_operations/ibm_operations.py (hunk 5 above)
Comment (athiruma) [reply cloud_governance/ibm/ibm_operations/ibm_operations.py:5]: done

File: cloud_governance/ibm/ibm_operations/tag_resources.py (hunk 6 above)
Comment (athiruma) [reply cloud_governance/ibm/ibm_operations/tag_resources.py:8]: done

File: cloud_governance/ibm/tagging/tagging_operations.py (hunk 7 above)
Comment (athiruma) [reply cloud_governance/ibm/tagging/tagging_operations.py:8]: done

File: cloud_governance/policy/ibm/tag_baremetal.py (hunk 3 above)
Comment (athiruma) [reply cloud_governance/policy/ibm/tag_baremetal.py:1]: done

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**